guides/source/getting_started.md

```diff
@@ -525,7 +525,6 @@ method called `form_with`. To use this method, add this code into
 ```
 
 If you refresh the page now, you'll see the exact same form from our example above.
-Building forms in Rails is really just that easy!
```

Comment:
Wouldn't it make more sense to keep this as "Building forms in Rails is really easy!" I've seen how other technologies are used to build forms and I can *guarantee you* that it's definitely a lot easier to build forms in Rails than in any other framework out there. We could even say that "Building forms in Rails is fast and straightforward".

Reply:
It's not easy if you have never done any programming before and the audience of a Getting Started page is more likely to be new programmers than seasoned switchers from another framework. ref: https://justsimply.dev

Reply:
Maybe it would be more precise to say that it needs less typing. A lot of the Magic of Rails™️ amounts to not having to type 500 characters where 50 would do.